app/src/main/java/org/mifos/mobilebanking/utils/MFDatePicker.java

```diff
@@ -28,7 +28,13 @@
     static Calendar calendar;
     private String startDateString;
     private long startDate;
-    private boolean isStartOrEnd;
+    private int datePickerType = ALL_DAYS;
+
+    /* Constants used to select which type of date picker is being called */
+    public static final int ONLY_CURRENT_DAY = 0; // only current day can be picked
```

Comment:
This is not needed, as if no date is to be selected then why to show date picker.